MonoGameProj/MonoGameProj/Game1.cs

```diff
@@ -12,20 +16,26 @@ public class Game1 : Game
         private GraphicsDeviceManager graphics;
         private SpriteBatch spriteBatch;
         private DeltaTimeCalculator deltaTimeCalculator;
+        private List<Player> players;
 
         // Textures
-        private Texture2D player;
-        private Vector2 playerPosition = new Vector2(100, 100);
+        private Texture2D firstPlayerSprite;
```

Comment:
Should player texture be stored in the player class? May be better to wait for an assetLoader manager before doing that